pkg/action/cl001/ac001/explainer.go

```diff
@@ -2,8 +2,52 @@ package ac001
 
 import (
 	"context"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/giantswarm/microerror"
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
-	return "", nil
+	crs, err := newCRs("https://g8s.codename.region.aws.gigantic.io:443")
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	clusterCR, err := yaml.Marshal(crs.Cluster)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	awsClusterCR, err := yaml.Marshal(crs.AWSCluster)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	g8sControlPlaneCR, err := yaml.Marshal(crs.G8sControlPlane)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	awsControlPlaneCR, err := yaml.Marshal(crs.AWSControlPlane)
```

Comment:
I would have liked to "marshal indent" the CRs for more visual support in the explainer output but the `ghodds` package does not appear to support custom indentations like the native JSON packages.